suites/apis/googleDataAccessTest.js

```diff
@@ -0,0 +1,309 @@
+  Commons.setUserYaml(Commons.userAccessFiles.newUserAccessFile2);
+  bash.runJob('useryaml-job');
+
+  console.log('using saved google creds to access google bucket!! Save responses to check later');
+  // use Google's client libraries to attempt to read a controlled access file with the
+  // creds we just saved (based on the user's permissions)
+  // attempt to access a file in the bucket
+  user0AccessQA2Res = await google.getFileFromBucket(
+    fence.props.googleBucketInfo.QA.googleProjectId,
+    pathToCreds0KeyFile,
+    fence.props.googleBucketInfo.QA.bucketId,
+    fence.props.googleBucketInfo.QA.fileName
+  );
+  user0AccessTest2Res = await google.getFileFromBucket(
+    fence.props.googleBucketInfo.test.googleProjectId,
+    pathToCreds0KeyFile,
+    fence.props.googleBucketInfo.test.bucketId,
+    fence.props.googleBucketInfo.test.fileName
+  );
+  user1AccessQA2Res = await google.getFileFromBucket(
+    fence.props.googleBucketInfo.QA.googleProjectId,
+    pathToCreds1KeyFile,
+    fence.props.googleBucketInfo.QA.bucketId,
+    fence.props.googleBucketInfo.QA.fileName
+  );
+  user1AccessTest2Res = await google.getFileFromBucket(
+    fence.props.googleBucketInfo.test.googleProjectId,
+    pathToCreds1KeyFile,
+    fence.props.googleBucketInfo.test.bucketId,
+    fence.props.googleBucketInfo.test.fileName
+  );
+  user2AccessQA2Res = await google.getFileFromBucket(
+    fence.props.googleBucketInfo.QA.googleProjectId,
+    pathToCreds2KeyFile,
+    fence.props.googleBucketInfo.QA.bucketId,
+    fence.props.googleBucketInfo.QA.fileName
+  );
+  user2AccessTest2Res = await google.getFileFromBucket(
+    fence.props.googleBucketInfo.test.googleProjectId,
+    pathToCreds2KeyFile,
+    fence.props.googleBucketInfo.test.bucketId,
+    fence.props.googleBucketInfo.test.fileName
+  );
+
+  console.log('deleting temporary google credentials');
+  // call our endpoint to delete temporary creds
+  const deleteCreds0Res = await fence.complete.deleteTempGoogleCreds(
+    creds0Key, users.user0.accessTokenHeader);
+  const deleteCreds1Res = await fence.complete.deleteTempGoogleCreds(
+    creds1Key, users.user1.accessTokenHeader);
+  const deleteCreds2Res = await fence.complete.deleteTempGoogleCreds(
+    creds2Key, users.user2.accessTokenHeader);
+
+  console.log('test cleanup: deleting google service accounts from google');
+  const deleteServiceAccount0Res = await google.deleteServiceAccount(
+    tempCreds0Res.body.project_id, tempCreds0Res.body.client_email);
+  const deleteServiceAccount1Res = await google.deleteServiceAccount(
+    tempCreds1Res.body.project_id, tempCreds1Res.body.client_email);
+  const deleteServiceAccount2Res = await google.deleteServiceAccount(
+    tempCreds2Res.body.project_id, tempCreds2Res.body.client_email);
+
+  console.log('deleting temporary google credentials file');
+  fs.unlink(pathToCreds0KeyFile, function(err) {
+    if (err) throw err;
+    console.log(`${pathToCreds0KeyFile} deleted!`);
+  });
```

Comment:
you can also use the util function here `files.deleteFile(pathToCreds0KeyFile);`
we could add a log for success to both util functions